components/custom_status/date_time_input.tsx

```diff
@@ -125,50 +95,88 @@ const DateTimeInputContainer: React.FC<Props> = (props: Props) => {
             const dayWithTimezone = timezone ? moment.tz(day, timezone) : moment(day);
             handleChange(dayWithTimezone.startOf('day'));
         }
+        handlePopperClosed();
     };
 
     const handleTimeChange = useCallback((time: Date, e: React.MouseEvent) => {
         e.preventDefault();
         handleChange(moment(time));
+        focusTimeButton();
     }, [handleChange]);
 
     const currentTime = getCurrentMomentForTimezone(timezone).toDate();
-    const modifiers = {
-        today: currentTime,
+
+    const focusTimeButton = useCallback(() => {
+        document.dispatchEvent(new CustomEvent<A11yFocusEventDetail>(
+            A11yCustomEventTypes.FOCUS, {
+                detail: {
+                    target: timeButtonRef.current,
+                    keyboardOnly: true,
+                },
+            },
+        ));
+    }, []);
+
+    const handlePopperClosed = useCallback(() => {
+        setIsPopperOpen(false);
+        props.setIsDatePickerOpen?.(false);
+    }, []);
+
+    const handlePopperOpen = useCallback(() => {
+        setIsPopperOpen(true);
+        props.setIsDatePickerOpen?.(true);
+    }, []);
+
+    const formatDate = (date: Date): string => {
```

Comment:
If Luxon can do that, that'd be preferred. We don't have a specific ticket or epic for it yet, but we're trying to move towards Luxon instead of Moment

Reply:
Using Luxon now, also replaced this function in the dnd modal